napari/__init__.py

```diff
@@ -42,6 +44,8 @@
     'viewer': ['Viewer', 'current_viewer'],
 }
 
+_LOG_STREAM = _get_custom_log_stream()
```

Comment:
I think that custom log handler should be initialized in `napari.qt.qt_event_loop`, same as notification manager.

Reply:
But the logger should work regardless of qt right? Also headless it would be useful.

Reply:
As I understand, it is log handler, not logger. The current code means, that each time one import napari it starts storing all logs in memory, even without a clean way to show them.

Reply:
Yes... but isn't that what we want? Would be good to be able to get these logs programmatically as well. We can also discard old logs if we go above a certain size.

Reply:
do what? I'm not sure I follow ^^'